internal/hcs/schema2/plan9_share.go

```diff
@@ -12,7 +12,7 @@ package hcsschema
 type Plan9Share struct {
 	Name string `json:"Name,omitempty"`
 
-	//  The name by which the guest operation system can access this share, via  the aname parameter in the Plan9 protocol.
+	//  The name by which the guest operation system can access this share, via  the aname parameter in the Plan9 guestprotocol.
```

Comment:
Should this be changed? I don't see guestprotocol defined anywhere, and this file is autogenerated

Reply:
Derp. I must've search/replaced "protocol" everywhere. good catch